modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/TopLevelBuild.java

```diff
@@ -329,11 +334,23 @@ public Element getValidationGitHubMessageElement() {
 		return validationBuild.getGitHubMessageElement();
 	}
 
+	public boolean hasPullRequestObject() {
+		if (_pullRequestObject == null) {
+			return false;
+		}
+
+		return true;
+	}
+
 	@Override
 	public void setCompareToUpstream(boolean compareToUpstream) {
 		_compareToUpstream = compareToUpstream;
 	}
 
+	public void setPullRequestObject(String pullRequestURL) {
```

Comment:
Aside from the issue with the name of the method.. I'm not sure it should even exist. I feel like a TopLevelBuild should be able to discover the pull request URL (possibly from a build parameter?) and instantiate the pull request object. If you should find that you absolutely need this method, it should have a PullRequest as the parameter, not a String called pullRequestURL. Keep in mind that a pullRequestURL should be an instance of the class, URL, not a String.